src/routes/music/components/edit-music-id.svelte

```
@@ -31,8 +29,9 @@
 	import Trash from "@lucide/svelte/icons/trash";
 	import { Switch } from "$lib/components/ui/switch";
 	import { Label } from "$lib/components/ui/label";
-	import { useSession } from "$lib/session";
 	import { hasPermission } from "$lib/permissions";
+	import { deleteMusicId, editMusicId, listMusicIds } from "$lib/music/music.remote.js";
```

ESLint

[error] 'listMusicIds' is defined but never used.

(@typescript-eslint/no-unused-vars)